kuka_resources/urdf/common_sim_macro.ros2_control.xacro

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do we need this file? “common_macro” should be extended a bit to support simulators.

Reply:
removed duplicate